src/components/DataSubmissions/CreateDataSubmissionDialog.tsx

```diff
@@ -550,14 +571,30 @@ const CreateDataSubmissionDialog: FC<Props> = ({ onCreate }) => {
       </CreateSubmissionDialog>
 
       {orgOwnerOrSubmitter && (
-        <StyledButton
-          type="button"
-          onClick={handleOpenDialog}
-          sx={{ bottom: "30px", right: "50px" }}
-          disabled={!hasOrganizationAssigned}
-        >
-          Create a Data Submission
-        </StyledButton>
+        <StyledTooltipWrapper alignItems="center" justifyContent="flex-end">
+          <Tooltip
+            placement="top"
+            title="Your associated organization is inactive. You cannot create a data submission at this time."
+            open={undefined} // will use hoverListener to open
+            disableHoverListener={!userHasInactiveOrg}
+          >
+            <span>
+              <StyledButton
+                type="button"
+                variant="contained"
+                onClick={handleOpenDialog}
+                disabled={
+                  !hasOrganizationAssigned ||
+                  userHasInactiveOrg ||
+                  authStatus === AuthStatus.LOADING ||
+                  orgStatus === OrgStatus.LOADING
```

Comment:
Do we still need to check the Organization Context status? When navigating to the data submission page, it briefly renders as disabled because the organization list is still loading. 

```suggestion
                  authStatus === AuthStatus.LOADING
```

If it's needed still, you can dismiss this.

Reply:
Good catch! Was there from before the BE was ready. Replaced the orgStatus check to instead check when approvedStudies data is loading, otherwise tests will fail.